<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtemDeviceDialog</class>
 <widget class="QDialog" name="AtemDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>145</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>145</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>145</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New ATEM Mixer</string>
  </property>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>67</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>112</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>37</y>
     <width>275</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>IP address / Hostname</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddress">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>41</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>63</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Optional</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDeviceName">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>11</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeviceName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeviceIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Widgets.qrc">:/Graphics/Images/ServerBig.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonTestConnection">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>37</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Test Connection</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditDeviceName</tabstop>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>lineEditDescription</tabstop>
 </tabstops>
 <resources>
  <include location="Widgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>AtemDeviceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AtemDeviceDialog</receiver>
   <slot>addressChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDeviceName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AtemDeviceDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonTestConnection</sender>
   <signal>clicked()</signal>
   <receiver>AtemDeviceDialog</receiver>
   <slot>testConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressFinished()</slot>
  <slot>nameFinished()</slot>
  <slot>testConnection()</slot>
  <slot>okClicked()</slot>
  <slot>nameChanged(QString)</slot>
  <slot>addressChanged(QString)</slot>
 </slots>
</ui>
